Zappy_IA_Report/NN/Code/Tutorial3.py

```
@@ -0,0 +1,37 @@
+import tensorflow as tf
```

Ruff (0.11.9)

`tensorflow` imported but unused

Remove unused import: `tensorflow`

(F401)